docs/src/dev-docs/new-architecture.rst

```diff
@@ -56,9 +56,8 @@ In order to follow this, a new architectures has two define two classes
     val_datasets passed to the Trainer, as well as the dataset_info passed to the
     model.
 
-The ``ModelInterface`` is the main model class and must implement a
-``save_checkpoint()``, ``load_checkpoint()``  as well as a ``restart()`` and
-``export()`` method.
+The ``ModelInterface`` is the main model class and must implement the
+``load_checkpoint()``, ``restart()`` and ``export()`` methods.
```

Comment:
`load_checkpoint` is also a trainer method now.

Reply:
Yes, and it's documented in the `TrainerInterface` just below